src/components/analyses/landing/AnalysisSubmissionLanding.js

```diff
@@ -0,0 +1,620 @@
+                                ].includes(analysis?.status) && (
+                                        <DataPathLink
+                                            id={baseId}
+                                            param_type="FolderInput"
+                                            path={analysis?.resultfolderid}
+                                        />
+                                    )}
+                            </div>
+                            <div style={{ marginLeft: theme.spacing(0.25) }}>
+                                <CopyLinkButton
+                                    baseId={baseId}
+                                    onCopyLinkSelected={() => {
+                                        const link = `${getHost()}/${NavigationConstants.DATA
+                                            }/${constants.DATA_STORE_STORAGE_ID}${analysis?.resultfolderid
+                                            }`;
+                                        const copyPromise =
+                                            copyStringToClipboard(link);
+                                        copyLinkToClipboardHandler(
+                                            i18nCommon,
+                                            copyPromise
+                                        );
+                                    }}
+                                />
+                            </div>
+                        </div>
+                    </GridLabelValue>
+                    <GridLabelValue label={t("startDate")}>
+                        <InfoGridValue>
+                            {formatDate(analysis?.startdate)}
+                        </InfoGridValue>
+                    </GridLabelValue>
+                    <GridLabelValue label={t("endDate")}>
+                        <InfoGridValue>
+                            {formatDate(analysis?.enddate)}
+                        </InfoGridValue>
+                    </GridLabelValue>
+                    <GridLabelValue label={t("user")}>
+                        <InfoGridValue>{username}</InfoGridValue>
+                    </GridLabelValue>
+                </Grid>
+                <Accordion
+                    expanded={expandHistory}
+                    onChange={() => setExpandHistory(!expandHistory)}
+                >
+                    <AccordionSummary
+                        expandIcon={<ExpandMore />}
+                        aria-controls={buildID(
+                            baseId,
+                            ids.STATUS_HISTORY_PANEL
+                        )}
+                        id={buildID(baseId, ids.STATUS_HISTORY_PANEL_HEADER)}
+                    >
+                        <Typography variant="subtitle2" color="primary">
+                            {t("statusHistory")}
+                        </Typography>
+                    </AccordionSummary>
+                    <div style={{ padding: theme.spacing(1) }}>
```

Comment:
I'm guessing `AccordionDetails` already sets some appropriate `padding`.
Also, should this be the panel with the `STATUS_HISTORY_PANEL` ID?
```suggestion
                    <AccordionDetails id={buildID(baseId, ids.STATUS_HISTORY_PANEL)}>
```